src/main/scala/com/atomist/rug/kind/core/ProjectMutableView.scala

```diff
@@ -416,11 +417,16 @@ class ProjectMutableView(
   }
 
   @ExportFunction(readOnly = false,
+    description = "Don't use. Merely intended to simplify the life of the Rug to TypeScript transpiler.")
+  def projects: java.util.List[ProjectMutableView] = Collections.singletonList(this)
+      
+  @ExportFunction(readOnly = false,
     exposeAsProperty = true,
     description = "Files in this project")
-  def files: java.util.List[FileMutableView] = {
```

Comment:
The return type is  java.util.List[FileArtifactBackedMutableView]. It wont compile with java.util.List[FileMutableView] as it's overriding method in ProjectView